docs/developer-docs/docs/build-an-agent/warden-agent-kit/agent-actions.md

```diff
@@ -4,19 +4,32 @@ sidebar_position: 3
 
 # Agent Actions
 
-By default, the Warden Agent Kit supports the following [Actions](/learn/glossary#action):
+## Actions on Warden
 
-- `get_spaces`: Retrieve all your [Spaces](/learn/glossary#space).
+An [Action](/learn/glossary#action) is any transaction on Warden Protocol. To perform an Action, one can use the [SpaceWard](https://help.wardenprotocol.org) application, run [node commands](/operate-a-node/node-commands), or prompt a Warden Agent.
+
+Each new user needs to perform the following Actions: get test [WARD tokens](/learn/glossary#ward-token), create a [Space](/learn/glossary#space), and request a [key](/learn/glossary#key) from a [Keychain](/learn/glossary#keychain). After doing it, the user will be able to send tokens, create [Orders](/learn/glossary#order), and initiate other Actions.
+
+:::note
+A Space is a hub allowing its owner to manage multiple cryptographic keys that identify users and secure the ownership of wallets. To create a key, a user requests it from a Keychain—a custodian that generates and stores keys and signs transactions with them.
+:::
+
+## Supported Agent Actions
+
+By default, the Warden Agent Kit supports the following Actions and queries:
+
+- `get_spaces`: Retrieve all your Spaces.
 - `create_space`: Create a new Space.
-- `get_keys`: Get all [keys](/learn/glossary#key) for a given Space.
-- `create_key`: Create a new key for a specified Space and [Keychain](/learn/glossary#keychain).
+- `get_keys`: Get all keys for a given Space.
+- `create_key`: Create a new key for a specified Space and Keychain.
 - `get_keychains`: Retrieve all available Keychains.
 - `get_balance`: Check the balance of Sepolia ETH for a specific key.
-- `request_funds`: Request [WARD tokens](/learn/glossary#ward-token) from the faucet.
+- `request_funds`: Request WARD tokens from the faucet.
```

Comment:
Should we add a quick note here saying temporarily unavailable? Message us on discord to get funds?